docs/_static/auth/v1.yaml

```diff
@@ -453,17 +453,21 @@ components:
           type: string
           description: |
             Base64 encoded IRMA contract conaining the identity of the performer
-        scope:
+        service:
           type: string
-          description: Space-delimited list of strings. For what kind of operations can the access token be used? Scopes will be specified for each use-case
-          example: nuts-sso nuts-patient-transfer
+          description: For which service can the access token be used? The right endpoint is selected based on the service.
```

Comment:
Question mark looks a bit strange in the description (it isn't a question)

Reply:
rewrote